boost_manager/tests/updater_tests.rs

```diff
@@ -124,14 +138,16 @@ async fn test_process_activations_failure(pool: PgPool) -> anyhow::Result<()> {
         Duration::from_secs(10),
         10,
         solana_connection,
+        Duration::from_millis(10),
     )?;
 
     let mut txn = pool.begin().await?;
     seed_activations(&mut txn, now).await?;
     txn.commit().await?;
 
-    // ensure the activations are processed at least 10 times
+    // process the activations
     // submit_txn will bork each time
+    // a failing txn will by default be retried 10 times
```

Comment:
Making sure I understand.
An activation will be processed 10 times before being marked as failed.
And a txn for an activation will be retried 10 times by default.
So this test is "submitting" 110 txns to get the activations into a failed state?

Reply:
There's a few things I don't know. 
- How long does a blockhash stay valid?
- How often activations are checked for submission (default looks to be 15mins)

Is it possible for an activation to get stuck in a loop to where it never gets marked as Failed in the DB?
It gets processed, but errors, and before the max retries in the db the blockhash cannot be found, so that set of retries gets reset. 
Rinse, repeat.

Reply:
Yes, an activation will be processed 10 times before being marked as failed.  Each activation attempt the solana txn can be retried up to 10 times.

A blockhash is valid for up to 150 blocks

I believe in mainnet the default activation tick is 15 mins.

I dont see potential for an activation to get stuck in a loop but I will give it a fresh look.  Is there something in particular that suggested to you the potential for that to occur ?